apps/explorer/lib/explorer/chain/import/runner/blocks.ex

```diff
@@ -124,18 +127,18 @@ defmodule Explorer.Chain.Import.Runner.Blocks do
   @impl Runner
   def timeout, do: @timeout
 
-  #  defp acquire_contract_address_tokens(repo, consensus_block_numbers) do
-  #    query =
-  #      from(ctb in Address.CurrentTokenBalance,
-  #        where: ctb.block_number in ^consensus_block_numbers,
-  #        select: ctb.token_contract_address_hash,
-  #        distinct: ctb.token_contract_address_hash
-  #      )
-  #
-  #    contract_address_hashes = repo.all(query)
-  #
-  #    Tokens.acquire_contract_address_tokens(repo, contract_address_hashes)
-  #  end
+  defp acquire_contract_address_tokens(repo, consensus_block_numbers) do
```

Comment:
I think this was commented out because on Celo blocks there are no blocks without consensus. Other methods in this file need to be checked agains both our and upstream versions.